sdk-extensions/autoconfigure/README.md

```diff
@@ -65,6 +65,7 @@ The [Jaeger](https://www.jaegertracing.io/docs/1.21/apis/#protobuf-via-grpc-stab
 |-----------------------------------|-----------------------------------|----------------------------------------------------------------------------------------------------|
 | otel.traces.exporter=jaeger       | OTEL_TRACES_EXPORTER=jaeger       | Select the Jaeger exporter                                                                         |
 | otel.exporter.jaeger.endpoint     | OTEL_EXPORTER_JAEGER_ENDPOINT     | The Jaeger gRPC endpoint to connect to. Default is `http://localhost:14250`.                       |
+| otel.exporter.jaeger.timeout      | OTEL_EXPORTER_JAEGER_TIMEOUT      | The maximum waiting time allowed to send each batch. Default is `10000`.                           |
```

Comment:
Perhaps adding unit (millis) here (and for OTLP readme) would be helpful for end-users (even though it's a common unit in similar contexts).